cmd/reconciler/main.go

```diff
@@ -75,10 +94,8 @@ var (
 		"The rate of updates per minute to an API Resource at which the Syncer logs warnings about too many updates to the resource.")
 	resyncPeriod = flag.Duration("resync-period", configsync.DefaultReconcilerResyncPeriod,
 		"Period of time between forced re-syncs from source (even without a new commit).")
-	workers = flag.Int("workers", 1,
-		"Number of concurrent remediator workers to run at once.")
 	pollingPeriod = flag.Duration("filesystem-polling-period",
-		controllers.PollingPeriod(reconcilermanager.ReconcilerPollingPeriod, configsync.DefaultReconcilerPollingPeriod),
+		rmcontrollers.PollingPeriod(reconcilermanager.ReconcilerPollingPeriod, configsync.DefaultReconcilerPollingPeriod),
```

Comment:
Why are we using a function from the reconciler-manager in the reconciler? 
Is it just re-use because they have the same flag and behavior?